data-prepper-core/src/main/java/org/opensearch/dataprepper/parser/model/PipelineConfiguration.java

```diff
@@ -116,6 +117,11 @@ private static PluginSetting getPluginSettingFromPluginModel(final PluginModel p
         return new PluginSetting(pluginModel.getPluginName(), settingsMap == null ? new HashMap<>() : settingsMap);
     }
 
+    private static RoutedPluginSetting getRoutedPluginSettingFromSinkModel(final SinkModel sinkModel) {
+        final Map<String, Object> settingsMap = sinkModel.getPluginSettings();
+        return new RoutedPluginSetting(sinkModel.getPluginName(), settingsMap == null ? new HashMap<>() : settingsMap, sinkModel.getRoutes());
```

Comment:
nit: The inline operator argument could be moved to its own line.

Another alternative is Optional
```
    public static String bar(final String arg) {
        return arg;
    }

    public static void main(String[] args) {
        final var foo = Optional.ofNullable(bar(args[0]));
        System.out.println(foo.orElse("default"));
    }
```